Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -85,22 +85,69 @@ LogManager::LogManager()
 
 	IniFile ini;
 	ini.Load(File::GetUserPath(F_LOGGERCONFIG_IDX));
+	IniFile::Section* options = ini.GetOrCreateSection("Options");
 	IniFile::Section* logs = ini.GetOrCreateSection("Logs");
+
+	// Set up log listeners
+	int verbosity;
+	options->Get("Verbosity", &verbosity, 0);
+
+	// Ensure the verbosity level is valid
+	if (verbosity < 1)
+		verbosity = 1;
+	if (verbosity > MAX_LOGLEVEL)
+		verbosity = MAX_LOGLEVEL;
+
+	// Get the logger output settings from the config ini file.
+	bool m_writeFile, m_writeConsole;
```

Comment:
Also, I don't think DolphinWX's `m_writeFile` and `m_writeConsole` are used anymore, so you can remove the declarations of those.